manifests/init.pp

```diff
@@ -500,6 +503,7 @@
   Optional[Stdlib::Absolutepath] $conf_enabled                               = $apache::params::conf_enabled,
   Stdlib::Absolutepath $vhost_dir                                            = $apache::params::vhost_dir,
   Optional[Stdlib::Absolutepath] $vhost_enable_dir                           = $apache::params::vhost_enable_dir,
```

Comment:
Do you think we could change this to `vhost_dir` to make the parameters less ambiguous? 

I'm also assuming that `vhost_enable_dir` in this instance is referring to the document root in the apache config?

Reply:
That would make this a backwards incompatible change and require a major release done.
We could add it as something to be done for the next major release though in the list pr:
https://github.com/puppetlabs/puppetlabs-apache/issues/2291

Reply:
Also, there already is a vhost_dir

Reply:
I think it would certainly need cleaning up yeah.

To me, `vhost_enable_dir` and `enable_vhost_enable_dir` is very confusing.

What is stopping us from cutting a major release?